docs/developers/build-starrocks/build_starrocks_on_ubuntu.md

```diff
@@ -0,0 +1,70 @@
+### Prerequisite
+```
+sudo apt-get install automake binutils-dev bison byacc ccache cmake flex libiberty-dev libtool maven zip -y
```

Comment:
If the versions of CMAKE and GCC are too low, they will not compile successfully. You should write the basic requirements or check in the script.

Reply:
cmake 3.16.3, or gcc 9.4. Larger than them?

Reply:
For me, the version I compiled myself is cmake 3.20.3, and gcc is 10.3.0. Change the script and compile it successfully. The actual minimum requirements are not very clear, but I remember that the cmake version that comes with ubuntu is definitely not good.